Sources/Authentication/LoginSessionConfiguration.swift

```diff
@@ -38,7 +38,7 @@ public struct LoginSessionConfiguration {
                 clientID: String,
                 prefersEphemeralWebSession: Bool = true,
                 redirectURI: String,
-                viewThroughRate: String = "[Cl.Cm.P0]",
+                vectorsOfTrust: String = "[\"Cl.Cm.P0\"]",
```

Comment:
should this be an array of strings instead? `vectorsOfTrust: [String] = ["Cl.Cm.P0"],`

Reply:
Looks as though the `OIDAuthorizationRequest`'s `additionalParameters` property expects a dictionary with values of `String`